packages/runtime-tools-swf-enveloped-components/src/workflowDefinitions/envelope/components/WorkflowDefinitionListToolbar/WorkflowDefinitionListToolbar.tsx

```diff
@@ -109,6 +109,7 @@ const WorkflowDefinitionListToolbar: React.FC<WorkflowDefinitionListToolbarProps
               onKeyPress={onEnterClicked}
               placeholder={`Filter by workflow name`}
               value={workflowNameInput}
+              data-testid="workflowFilterInput"
```

Comment:
Extra small suggestion to follow the repository style for `data-testid`
```suggestion
              data-testid="workflow-filter-input"
```